automation/test.sh

```diff
@@ -29,15 +29,15 @@ readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluste
 
 # Skip if it's docs changes only
 # Only if we are in CI, and this is a non-batch change
-if [[ ${CI} == "true" && -n "$PULL_BASE_SHA" && -n "$PULL_PULL_SHA" ]]; then
+if [[ ${CI} == "true" && -n "$GITHUB_TOKEN" && -n "$PULL_NUMBER" ]]; then
     SKIP_PATTERN="^(docs/)|(OWNERS|OWNERS_ALIASES|.*\.(md|txt))$"
-    CI_GIT_ALL_CHANGES=$(git diff --name-only $PULL_BASE_SHA $PULL_PULL_SHA)
```

Comment:
Referring to the previous code
I remember vaguely that maybe `diff PULL_BASE_SHA..PULL_PULL_SHA` can help in this case ?
(to add the ..) not sure, but if you want to try

Reply:
Thanks Or - you pointed me in the right direction

Reply:
can you please explain why `...` and not `..` ?
(`...` is the common ancestor of A...B something like that)

unrelated nit: `.*\.(md|txt)` -> `\.(md|txt)`, no need the `.*` i think